flask_ask/core.py

```diff
@@ -608,12 +608,15 @@ def _flask_view_func(self, *args, **kwargs):
             else:
                 result = "{}", 200
         elif request_type == 'IntentRequest' and self._intent_view_funcs:
-            result = self._map_intent_to_view_func(self.request.intent)()
+            try:
+                result = self._map_intent_to_view_func(self.request.intent)()
+            except Exception as e:
+                print("Error:", e)
+                exit()
```

Comment:
There are a couple of issues with this code:

1. The Exception caught shouldn't be general
1. print() should not be used in production code
1. exit() should not be called.